src/main/twirl/com/lightbend/lagom/docs/faq.scala.html

```diff
@@ -0,0 +1,49 @@
+@import com.lightbend.lagom.docs.LagomContext
+
+@(implicit context: LagomContext)
+
+@page("FAQ") {
+  <div>
+    <h1>Frequently Asked Questions</h1>
+    <blockquote>
+    Have a question about Lagom? We've sorted out a sample of frequently asked questions and provided answers below. If your question is not covered here, try these resources:
+    <ul>
+      <li><a href="//gitter.im/lagom/lagom">Lagom Gitter channel</a>
+      <li><a href="//groups.google.com/forum/#!forum/lagom-framework">lagom-framework mailing list</a> </li>
+      <li><a href="//stackoverflow.com/questions/tagged/lagom">Stackoverflow Lagom questions.</a></li>
+    </blockquote>
```

Comment:
This was the only way I could get the first paragraph not to outdent. It might be a problem on my system. I'll remove the blockquote and see what happens.